va/va.h

```diff
@@ -3922,6 +3922,38 @@ VAStatus vaQuerySurfaceError(
  */
 #define VA_FOURCC_IYUV          0x56555949
 
+/** Y8: 8-bit greyscale.
+ */
+#define VA_FOURCC_Y8            0x20203859
+
+/** Y16: 16-bit greyscale.
+ */
+#define VA_FOURCC_Y16           0x20363159
+
+/** VYUV: packed 8-bit YUV 4:2:2.
+ *
+ * Four bytes per pair of pixels: V, Y, U, V.
+ */
+#define VA_FOURCC_VYUY          0x59555956
+
+/** YVYU: packed 8-bit YUV 4:2:2.
+ *
+ * Four bytes per pair of pixels: Y, V, Y, U.
+ */
+#define VA_FOURCC_YVYU          0x55595659
+
+/** ARGB: packed 16-bit BGRA.
+ *
+ * Four bytes per pixel: alpha, red, green, blue.
+ */
+#define VA_FOURCC_ARGB64        0x34475241
```

Comment:
Is it possible for us to describe the bits for every channel? VA_FOURCC_ARGB16161616 which is following the above naming rules? Then, the name rules seems to be consistent. Thanks a lot in advance!

Reply:
change the command following the reviewed